src/api/test/functional/source_controller_test.rb

```diff
@@ -785,7 +785,7 @@ def test_lock_project
     assert_response 403
     post '/build/home:Iggy?cmd=wipe'
     assert_response 403
-    post '/build/home:Iggy/TestLinkPack?cmd=wipe'
+    post '/source/home:Iggy/TestLinkPack?cmd=wipe'
```

Comment:
these are here for backward compability reasons. Old remote OBS instance still may use them

Reply:
> these are here for backward compability reasons. Old remote OBS instance still may use them

AFAIK, the backend never supported the `cmd` parameter in this GET line: https://github.com/openSUSE/open-build-service/blob/7ffd60ad56d87ee52d6816503eb23fae36109c3b/src/backend/bs_srcserver#L7725

Neither has supported paths like `POST:/build/$project/$repository cmd:`.

If this route ever existed in the backend and later on was removed, keeping a test for compability only for the permission case doesn't make much sense. Either tests are provided to test full compability, or they are removed.